layouts/shortcodes/appsec-getstarted-2.md

```diff
@@ -0,0 +1,26 @@
+   The library collects security data from your application and sends it to the Agent, which sends it to Datadog, where [out-of-the-box detection rules][202] flag attacker techniques and potential misconfigurations so you can take steps to remediate. 
+   
+1.  **To see Application Security threat detection in action, send known attack patterns to your application**. The *Security Scanner Detected* rule features three cases:
```

Comment:
I feel the rule needs a greater introduction. Here it feels a bit dropped without context

Reply:
Yeah, I was thinking I would hunt down a link to the rule, when it's available. I'll leave this comment open until I get a chance to do that.